File: QB_Challenges/PrepInsta/JAVA/03_Sum_of_N_Numbers/SumOfN_formulae.java
import java.util.Scanner;

public class SumOfN_formulae {

    public static void SONN() {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the number: ");
        int num = sc.nextInt();
        sc.close();
        
        // formulae
        int sum = (num*(num+1)/2);

        System.out.println("the sum of first "+num+" numbers is: "+sum);
        
    }

    public static void main(String[] args) {
        SONN();
    }
    
}

File: QB_Challenges/PrepInsta/CPP/03_Sum_of_N_Numbers/SumOfN_formulae.cpp
#include <iostream>
using namespace std;

void SONN(){
    int num;
    cout <<"Enter the number: ";
    cin >>num;

    int sum = (num*(num+1)/2);

    cout <<"The sum of 0 to "<<num<<" numbers is: "<< sum <<endl;
}

int main(){
    SONN();
    return 0;
}